src/main/resources/db/migration/V116__sletter_blankettbehandlinger.sql

```diff
@@ -0,0 +1,174 @@
+DELETE
+FROM soknad_soker
+WHERE grunnlag_soknad_id IN (SELECT soknad_grunnlag.id
+                             FROM soknad_grunnlag
+                                      JOIN behandling ON soknad_grunnlag.behandling_id = behandling.id
+                                 AND behandling.type = 'BLANKETT');
+DELETE
+FROM vedlegg
+WHERE grunnlag_soknad_id IN (SELECT sg.id
+                             FROM soknad_grunnlag sg
+                                      JOIN behandling b ON b.id = sg.behandling_id
+                                 AND b.type = 'BLANKETT');
+DELETE
+FROM soknad_selvstendig
+WHERE soknadsskjema_id IN (SELECT s.id
+                           FROM soknadsskjema s
+                                    JOIN soknad_grunnlag sg ON s.id = sg.soknadsskjema_id
+                                    JOIN behandling b ON b.id = sg.behandling_id
+                               AND b.type = 'BLANKETT');
+DELETE
+FROM soknad_arbeidsgiver
+WHERE soknadsskjema_id IN (SELECT s.id
+                           FROM soknadsskjema s
+                                    JOIN soknad_grunnlag sg ON s.id = sg.soknadsskjema_id
+                                    JOIN behandling b ON b.id = sg.behandling_id
+                               AND b.type = 'BLANKETT');
+DELETE
+FROM soknad_aksjeselskap
+WHERE soknadsskjema_id IN (SELECT s.id
+                           FROM soknadsskjema s
+                                    JOIN soknad_grunnlag sg ON s.id = sg.soknadsskjema_id
+                                    JOIN behandling b ON b.id = sg.behandling_id
+                               AND b.type = 'BLANKETT');
+DELETE
+FROM soknad_utenlandsopphold
+WHERE soknadsskjema_id IN (SELECT s.id
+                           FROM soknadsskjema s
+                                    JOIN soknad_grunnlag sg ON s.id = sg.soknadsskjema_id
+                                    JOIN behandling b ON b.id = sg.behandling_id
+                               AND b.type = 'BLANKETT');
+DELETE
+FROM soknad_tidligere_utdanning
+WHERE soknadsskjema_id IN (SELECT s.id
+                           FROM soknadsskjema s
+                                    JOIN soknad_grunnlag sg ON s.id = sg.soknadsskjema_id
+                                    JOIN behandling b ON b.id = sg.behandling_id
+                               AND b.type = 'BLANKETT');
+DELETE
+FROM soknad_barnepassordning
+WHERE barn_id IN (SELECT b.id
```

Comment:
b.id her er behandlingen sitt ID. Hadde man kunnet brukt temporary view for å finne de som skal fjernes? (Usikker på om allt her er riktig)
eks 
```sql
CREATE TEMPORARY VIEW blankett_behandlinger AS
SELECT id
FROM behandling
WHERE type = 'BLANKETT';

CREATE TEMPORARY VIEW blankett_soknad_grunnlag AS
SELECT id, soknadsskjema_id
FROM soknad_grunnlag
WHERE behandling_id IN (SELECT id FROM blankett_behandlinger);

CREATE TEMPORARY VIEW blankett_soknadsskjema AS
SELECT id
FROM soknadsskjema
WHERE id IN (SELECT soknadsskjema_id FROM blankett_soknad_grunnlag);

DELETE
FROM soknad_soker
WHERE grunnlag_soknad_id IN (SELECT id FROM blankett_soknad_grunnlag);
DELETE
FROM vedlegg
WHERE grunnlag_soknad_id IN (SELECT id FROM blankett_soknad_grunnlag);
DELETE
FROM soknad_selvstendig
WHERE soknadsskjema_id IN (SELECT id FROM blankett_soknadsskjema);
DELETE
FROM soknad_arbeidsgiver
WHERE soknadsskjema_id IN (SELECT id FROM blankett_soknadsskjema);
DELETE
FROM soknad_aksjeselskap
WHERE soknadsskjema_id IN (SELECT id FROM blankett_soknadsskjema);
DELETE
FROM soknad_utenlandsopphold
WHERE soknadsskjema_id IN (SELECT id FROM blankett_soknadsskjema);
DELETE
FROM soknad_tidligere_utdanning
WHERE soknadsskjema_id IN (SELECT id FROM blankett_soknadsskjema);
DELETE
FROM soknad_barnepassordning
WHERE barn_id IN (SELECT id
                  FROM soknad_barn
                  where soknadsskjema_id in (SELECT id FROM blankett_soknadsskjema));
DELETE
FROM soknad_barn
WHERE (soknadsskjema_id) IN (select id from blankett_soknadsskjema);

DELETE
FROM soknadsskjema
WHERE id IN (select id from blankett_soknadsskjema);

DELETE
FROM soknad_grunnlag
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM behandling_ekstern
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM oppgave
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM vilkarsvurdering
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM behandlingsjournalpost
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM andel_tilkjent_ytelse
WHERE tilkjent_ytelse IN (SELECT t.id
                          FROM tilkjent_ytelse t
                                   JOIN blankett_behandlinger b ON b.id = t.behandling_id);
DELETE
FROM tilkjent_ytelse
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM behandlingshistorikk
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM blankett
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM registergrunnlag
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM vedtak
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM grunnlagsdata
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM vedtaksbrev
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM mellomlagret_brev
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM simuleringsresultat
WHERE behandling_id IN (SELECT id from blankett_behandlinger);

DELETE
FROM tilbakekreving
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM mellomlagret_fritekstbrev
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM brevmottakere
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM behandling_barn
WHERE behandling_id IN (SELECT id from blankett_behandlinger);
DELETE
FROM behandling
WHERE behandling.type = 'BLANKETT';
```

Reply:
Kommenterte ut de som ikke finnes i prod

Reply:
Jeg begynte med det, men synes det ble ryddigere uten, så jeg har egentlig akkurat skrevet meg vekk fra det.